data/json/items/tool/entry_tools.json

```
@@ -21,6 +21,22 @@
     "techniques": [ "WBLOCK_1" ],
     "flags": [ "BELT_CLIP" ]
   },
+  {
+    "id": "hairpin_picklock",
+    "type": "GENERIC",
+    "category": "tools",
+    "looks_like": "hairpin",
+    "name": { "str": "hairpin lockpick" },
+    "description": "This is a metal hairpin bent into shape by hand, worse than an improvised set but easier to work with than a bone skewer.  You need MacGyver-like skills to open locks with this as it is brittle, but it lowers the chance of an alarm being set off.",
```

And now, for consistency with the other picks... :3
```suggestion
    "description": "This is a metal hairpin bent into shape by hand, worse than an improvised set but easier to work with than a bone skewer.  You need MacGyver-like skills to open locks with this as it is brittle, but it lowers the chance of an alarm being set off.  If you're crouched, examining a locked door will have you prioritize picking the lock instead of using a crowbar on it.",
```